Editor/Module Management/ModuleUpdater.cs

```diff
@@ -0,0 +1,111 @@
+                .Select(PackageInfo.FindForAssetPath)
+                .ToList();
+
+            if (packages.Count == 0)
+            {
+                Debug.LogWarning($"No Ready Player Me packages found.");
+            }
+
+            foreach (var package in packages)
+            {
+                var repoUrl = package.packageId.Split('@')[1];
+                var releasesUrl = repoUrl.Replace(GITHUB_WEBSITE, GITHUB_API_URL)
+                                              .Split(new[] { ".git#" }, StringSplitOptions.None)[0] + "/releases";
+                var packageUrl = repoUrl.Split('#')[0];
+                var version = package.version.Split('-')[0];
+                FetchReleases(package.name, packageUrl, releasesUrl, new Version(version));
+            }
+        }
+
+        private static async void FetchReleases(string packageName, string packageUrl, string releasesUrl,
+            Version currentVersion)
+        {
+            UnityWebRequest request = UnityWebRequest.Get(releasesUrl);
+            UnityWebRequestAsyncOperation op = request.SendWebRequest();
+            while (!op.isDone) await Task.Yield();
+
+            if (request.result == UnityWebRequest.Result.Success)
+            {
+                var response = request.downloadHandler.text;
+                Release[] releases = JsonConvert.DeserializeObject<Release[]>(response);
+
+                Version[] versions = releases.Select(r => new Version(r.Tag.Substring(1).Split('-')[0])).ToArray();
+
+                Version latestVersion = versions.Max();
+
+                if (latestVersion > currentVersion)
+                {
+                    PromptForUpdate(packageName, currentVersion, latestVersion, packageUrl);
+                }
+            }
+            else
+            {
+                Debug.Log($"Failed to fetch { packageName } releases. Error: {request.error} ");
+            }
+        }
+
+        private static void PromptForUpdate(string packageName, Version currentVersion, Version latestVersion,
+            string packageUrl)
+        {
+            bool shouldUpdate = EditorUtility.DisplayDialog("Update Packages",
+                $"New update available for {packageName}\nCurrent version: {currentVersion}\nLatest version: {latestVersion}",
+                "Update",
+                "Skip");
+
+            if (shouldUpdate)
+            {
+                packageUrl += "#v" + latestVersion;
+                Update(packageName, packageUrl, currentVersion, latestVersion);
+            }
+            else
+            {
+                // TODO: Bring analytics here
+                Debug.LogWarning("Update skipped.");
+            }
+        }
+
+        private static void Update(string name, string url, Version current, Version latest)
+        {
+            RemoveRequest removeRequest = Client.Remove(name);
+            while (!removeRequest.IsCompleted) Thread.Sleep(20);
```

Comment:
we use this Sleep(20) multiple times, perhaps we should store the 20 in a constant field for reuse?